R/subset_datarqst.R

```diff
@@ -37,66 +45,141 @@
 subset_datarqst <- function(file = file.choose(),
                             PopID = unique(pop_names$code),
                             Species = unique(Species_codes$Code),
-                            filter = NULL){
+                            filter = NULL,
+                            include_conflicting = FALSE,
+                            output_type = "R",
+                            return_R = FALSE,
+                            save = TRUE,
+                            test = FALSE){
+
+  if(!test){
+    readRDS(file = file)
```

Comment:
Should this say the following?
```suggestion
    standard_data <- readRDS(file = file)
```

Reply:
@StefanVriend as far as I understand, no it should not. The RDS file that is written by run_pipelines() and read in here is a workspace containing an object called "standard_data". So there should not be any need to assign that here.